htdocs/core/class/rssparser.class.php

```diff
@@ -135,7 +138,18 @@ public function getImageUrl()
      */
     public function getLink()
     {
-        return $this->_link;
+        // only check links without arguments
+        if(strpos($this->_link, "?") > 0 || strpos($this->_link, "=") > 0 || strpos($this->_link, "&") > 0)
+        {
+            return $this->_link;
+        }
+
+        $result = removeStringFromEnd($this->_link, "/rss/");
```

Comment:
Why cleaning the property ->link ? Do you have example of use case that need this ?

Reply:
Hello,

The news feed of `https://www.energiezukunft.eu/rss/`

use `https://www.energiezukunft.eu/rss/` as link inside the XML

Reply:
Sorry. I dont understand. If inside the XML we have the link https://www.energiezukunft.eu/rss/, so it means we must use this URL (to get RSS) and to swich to direct RSS view/download

Reply:
Oh 🤦 , I see my mistake, I have misinterpreted the button as "Go-to Website" (main page or blog site) instead of view/download the RSS feed.